cypress/e2e/registration.cy.js

```diff
@@ -2,10 +2,73 @@
 
 describe('Student Registration page', () => {
   before(() => {
-
+    cy.visit('https://demoqa.com/automation-practice-form');
+    cy.viewport(1920, 1080);
   });
 
-  it('', () => {
+  it('Automation registration form', () => {
```

Comment:
The test case should have a more descriptive name. Instead of 'Automation registration form' it may be 'Should successfully register a student and assert the data in a modal window'.